### DISCUSSION TITLE
In-game subtitles?

### DISCUSSION THREAD

**Discussion by PlanetaRob** [stohrendorf/CroftEngine#308] [Ideas]
Hello!

I would like to know if you plan to implement in-game subtitles for the dialogues. This game never came out in Spanish and I am currently working on translating it.

I have started with the videos and everything is perfect, but it is impossible for me to introduce subtitules in the in-game dialogues, since I don't know how to do it.

It would be great if this feature was made available so that it could be translated into as many languages ​​as possible.

**Comment by stohrendorf** [comment]
I'm actually thinking about subtitles for a few years already, but it was never important enough to even create a ticket for it, as the demand is very low.

**Reply by PlanetaRob** [reply]
If one day you manage to do them, count with my help for the subtitles in Spanish.
In a few days I will finish subtitling and traslating all videos and dialogues :)

**Reply by stohrendorf** [reply]
OpenLara already did this: https://github.com/XProger/OpenLara/blob/master/src/lang/es.h

**Reply by PlanetaRob** [reply]
I didn't know... In any case, OpenLara does not allow videos in .mp4, audio in .flac or HD textures.

I'm looking for the best possible graphic and audio quality to make a tutorial on my YouTube channel.
